clusters/app.ci/supplemental-ci-images/assisted-ci-tools/temp-images-for-migration.yaml

```diff
@@ -1,5 +1,58 @@
 kind: ImageStream
 apiVersion: image.openshift.io/v1
+metadata:
+  namespace: edge-infrastructure
+  name: ocm-2.8
+spec:
+  tags:
+  - name: assisted-installer
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-installer
+      namespace: edge-infrastructure
+  - name: assisted-installer-agent
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-installer-agent
+      namespace: edge-infrastructure
+  - name: assisted-installer-controller
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-installer-controller
+      namespace: edge-infrastructure
+  - name: assisted-service
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-service
+      namespace: edge-infrastructure
+  - name: assisted-image-service
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-image-service
+      namespace: edge-infrastructure
+  - name: assisted-test-infra
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-test-infra
+      namespace: edge-infrastructure
+  - name: cluster-api-provider-agent
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:cluster-api-provider-agent
+      namespace: edge-infrastructure
+  - name: assisted-service-operator-index
```

Comment:
I don't think we have this image anymore
We have ``assisted-service-index`` which should change its target name imho:
https://github.com/openshift/release/pull/32685#discussion_r982635167

cc @adriengentil

Reply:
yes, we can get rid of it.
We'll need to mirror the index image once `ocm-2.7` tag is pushed here: https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_edge_infrastructure